docs/Users_Guide/line.rst

```diff
@@ -103,13 +113,13 @@ For example:
 
 *plot_filename: /username/working_dir/output_plots/line.png*
 
-This is where */username/myworkspace/* is *$METPLOTPY_SOURCE* and
-*/username/working_dir* is *$WORKING_DIR*.  Make sure that the
+This is where */username/myworkspace/METplotpy* corresponds to *$METPLOTPY_BASE* and
+*/username/working_dir* corresponds to *$WORKING_DIR*.  Make sure that the
 *$WORKING_DIR* directory that is specified exists and has the appropriate
 read and write permissions.  The path listed for *plot_filename* may be
-changed to the output directory of one's  choosing.  If this is not set,
+changed to the output directory of one's choosing.  If this is not set,
 then the *plot_filename* setting specified in the
-*$METPLOTPY_SOURCE/METplotpy/metplotpy/plots/config/line_defaults.yaml*
+*$METPLOTPY_BASE/metplotpy/plots/config/line_defaults.yaml*
 configuration file will be used.
 
 To save the intermediate **.points1** file (used by METviewer and is useful
```

Comment:
@bikegeek, I do not see a points_path in the $METPLOTPY_BASE/test/line/custom_line.yaml file.